Public/Src/Engine/Scheduler/Tracing/Log.cs

```diff
@@ -145,6 +145,20 @@ internal static string PipWriteFileFailedMessage(string pipDescription, string p
             string reason,
             string message);
 
+        [GeneratedEvent(
+            (ushort)EventId.PipIpcFailedDueToInvalidInput,
+            EventGenerators = EventGenerators.LocalOnly,
+            EventLevel = Level.Error,
+            Keywords = (int)(Events.Keywords.UserMessage | Events.Keywords.UserError),
+            EventTask = (ushort)Events.Tasks.PipExecutor,
+            Message = "IPC operation '{operation}' could not be executed via IPC moniker '{moniker}'.  Reason: {reason}. Error: {message}")]

```

Comment:
This user message is exactly the same as for `PipIpcFailed`.  To reduce potential confusion, I would change this message to include something about "InvalidInput"